qcodes/tests/drivers/test_keithley_26xx.py

```diff
@@ -40,27 +42,85 @@ def test_smu_channels_and_their_parameters(driver):
 
         assert 'current' == smu.mode()
         smu.mode('voltage')
+        assert smu.mode() == 'voltage'
 
-        assert 'off' == smu.output()
-        smu.output('on')
+        assert 0 == smu.output()
```

Comment:
For all the parameters that use create_on_off_val_mapping, could you change the test code so that it asserts for `True`/`False`, for example, like this:
```suggestion
        assert True is smu.output()
```
This is needed so that when somebody reads the test it is clear that these parameters are intended for `True`/`False` values (and not for "0"/"0.0"/etc, although those also work because it's python and things like `True == 0.0` are true)